index.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express")
+const showdata = require("./showdata")
+
+const app = express()
+
+app.set("view engine", "pug")
```

Comment:
These ask you to only use single quotes lets be sure your lint is working sometime this week.